lookout/core/tests/server.py

```diff
@@ -7,37 +9,75 @@
 import subprocess
 import sys
 import tarfile
+from urllib.error import HTTPError
 from urllib.request import urlopen
 
-from lookout.core.api.version import __version__ as binver
 
+from lookout.core.api.version import __version__ as binver
 
 file = pathlib.Path(__file__).parent / "server"
 
 
-def fetch():
+def fetch() -> pathlib.Path:
+    """
+    Fetch corresponding lookout-sdk executable.
+
+    :return: Path to lookout-sdk executable
+    """
+    log = logging.getLogger("fetch")
     try:
         buffer = io.BytesIO()
         with urlopen("https://github.com/src-d/lookout/releases/download/"
-                     "%s/lookout-sdk_%s_linux_amd64.tar.gz" % (binver, binver)) as response:
+                     "%s/lookout-sdk_%s_%s_amd64.tar.gz" % (binver, binver, sys.platform)
+                     ) as response:
             copyfileobj(response, buffer)
         buffer.seek(0)
         with tarfile.open(fileobj=buffer, mode="r:gz") as tar:
             with file.open("wb") as fout:
                 copyfileobj(tar.extractfile("lookout-sdk_linux_amd64/lookout-sdk"), fout)
         os.chmod(str(file), 0o775)
-    except Exception as e:
+        return file
```

Comment:
`file` is already present in the global namespace. Let's not prematurely overcomplicate.